src/LuigisBoxBundle/Tests/Unit/Factory/ContentUpdateFactoryTest.php

```diff
@@ -0,0 +1,59 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Answear\LuigisBoxBundle\Tests\Unit\Factory;
+
+use Answear\LuigisBoxBundle\Factory\ContentUpdateFactory;
+use Answear\LuigisBoxBundle\Service\ConfigProvider;
+use Answear\LuigisBoxBundle\Service\LuigisBoxSerializer;
+use Answear\LuigisBoxBundle\ValueObject\ContentUpdate;
+use Answear\LuigisBoxBundle\ValueObject\ContentUpdateObjects;
+use Answear\LuigisBoxBundle\ValueObject\ObjectsInterface;
+use PHPUnit\Framework\TestCase;
+
+class ContentUpdateFactoryTest extends TestCase
+{
+    /**
+     * @test
+     */
+    public function prepareRequestSuccessfully(): void
+    {
+        $objects = $this->getObjects();
+        $factory = $this->getFactory($objects);
+
+        $request = $factory->prepareRequest($objects);
+
+        $this->assertSame('host//v1/content', $request->getUri()->getPath());
```

Comment:
double // ?

Reply:
Its by mistake. I'll fix it

Reply:
Fit at https://github.com/answear/luigis-box-bundle/pull/4/files#diff-3f1939ab2f8840dbb31bd46b78fabd0c